BUILD.bazel

```diff
@@ -104,11 +104,11 @@ py_binary(
 
 genrule(
     name = "build_version_inc",
-    srcs = ["CHANGES"],
     outs = ["build-version.inc"],
-    cmd = "SOURCE_DATE_EPOCH=0 $(location :update_build_version) $(location CHANGES) $(location build-version.inc)",
-    cmd_bat = "set SOURCE_DATE_EPOCH=0  && $(location :update_build_version) $(location CHANGES) $(location build-version.inc)",
+    cmd = "SOURCE_DATE_EPOCH=0 $(location :update_build_version) $(RULEDIR) $(location build-version.inc)",
+    cmd_bat = "set SOURCE_DATE_EPOCH=0  && $(location :update_build_version) $(RULEDIR) $(location build-version.inc)",
     exec_tools = [":update_build_version"],
+    local = True,
```

Comment:
Because local is set to true, I think `exec_tools` should be changed to `tools`. Otherwise, if we have a Mac host system that is running on a Linux RBE instance, it will try to use the Linux Python version on the Mac host.